relay-event-normalization/src/normalize/span/description/sql/parser.rs

```diff
@@ -181,10 +186,21 @@ impl VisitorMut for NormalizeVisitor {
                     v.rows = vec![vec![Expr::Value(Self::placeholder())]]
                 }
             }
-            // `UPDATE "foo"."bar"` becomes `UPDATE bar`.
+            // Simple lists of col = value assignments are collapsed to `..`.
             Statement::Update { assignments, .. } => {
-                for assignment in assignments.iter_mut() {
-                    Self::simplify_compound_identifier(&mut assignment.id);
+                if assignments.len() > 1
+                    && assignments
+                        .iter()
+                        .all(|a| matches!(a.value, Expr::Value(_)))
+                {
+                    *assignments = vec![Assignment {
+                        id: vec![Ident::new("___UPDATE_LHS___")],
+                        value: Expr::Value(Value::Null),
+                    }]
+                } else {
+                    for assignment in assignments.iter_mut() {
```

Comment:
When there's either 1 or 0 assignments, I'd just take it (instead of iterating).